lib/game.rb

```diff
@@ -42,30 +41,14 @@ def instructions_prompt
     'Enter a row letter and column number. For Example: "A1".'
   end
 
-  def get_user_coordinates
-    until valid_input?
-      prompt_for_new_coordinates
-    end
+  def play
+    GetValidMoveService.new(
+      std_in: std_in,
+      std_out: std_out
+    ).call
```

Comment:
For this exercise, I think we should avoid command objects like this. If we focus on the messages being passed around, "call" doesn't convey much meaning. Since we're getting a move here, it'd be nice for the method name to be based on that. With an updated method name, the class name becomes a little awkward, and probably changes as well.

Reply:
Very good point!